util/src/main/resources/SummaryReport.ftl

```diff
@@ -189,4 +195,15 @@
       </div>
     </#if>
   </body>
+  <script>
+    $(document).ready(function () {
+      $('#statvars-table').DataTable({
```

Comment:
do we want to do this for the series summary table for each place as well?

Reply:
I do not think that will be useful, since the values there are not really meant for comparison across series, but instead a description of the time-series within itself.

Reply:
I actually think thats a good idea.  We have imports with a large number of SVs for a place, and for the same reasons that we may want to sort the main table (by SV name, unit, #obs, etc.) we might want to sort this too.